lib/l10nUtil.js

```diff
@@ -138,11 +145,49 @@ module.exports.ethereumRemoteClientPaths = [
   '../../../ethereum-remote-client/brave/app/_locales/en/messages.json'
 ]
 
+// Helper functions that's, for a given pair of chromium to brave GRD mapping
+// from the supplied map, determines which GRDP parts are no longer present in
+// the chromium GRD file.
+async function getRemovedGRDParts(mapping) {
```

Comment:
Why does this function need to be async? It seems like the whole function block doesn't contain any async operations. Also, it already await all the promises in `rebaseBraveStringFilesOnChromiumL10nFiles`, you can return `getRemovedGRDParts` directly in `rebaseBraveStringFilesOnChromiumL10nFiles`  which will be wrapped as resolved promise and used by `await rebaseBraveStringFilesOnChromiumL10nFiles`

Reply:
Definitely doesn't need to be `async` and doesn't need to be `await`ed in `rebaseBraveStringFilesOnChromiumL10nFiles`. Fixed in f9d3458b8f7bca55616c3a652a1737f9b69cf6ff. 
I do need to call it in the beginning of the `rebaseBraveStringFilesOnChromiumL10nFiles`, though, because `rebaseBraveStringFilesOnChromiumL10nFiles` will overwrite brave files from chromium files and then I won't be able to figure out what got removed.